ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml

```diff
@@ -91,8 +91,8 @@
 # SLES
 - name: Download NVidia CUDA toolkit Repo RPM for SLES12 on x86_64
   get_url:
-    url="http://developer.download.nvidia.com/compute/cuda/repos/sles122/x86_64/cuda-repo-sles122-9.0.176-1.x86_64.rpm"
-    dest="/tmp/sles12_cuda9_repo.rpm"
+    url: http://developer.download.nvidia.com/compute/cuda/repos/sles122/x86_64/cuda-repo-sles122-9.0.176-1.x86_64.rpm
```

Comment:
Just noticed this - can we go https?

Reply:
Fixed, ready for your review, thank you!